webapp/src/mutator.ts

```diff
@@ -250,7 +250,7 @@ class Mutator {
         }
         const srcTemplate = newBoard.fields.cardProperties[index]
         const newTemplate: IPropertyTemplate = {
-            id: Utils.createGuid(),
+            id: Utils.createGuid(''),
```

Comment:
I think this ends up being a block, shouldn't it have the `block` type?

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c